files/build_templates/instance/syncd.service.j2

```diff
@@ -1,27 +1,48 @@
 [Unit]
 Description=syncd service
+{% if multi_instance == 'true' %}
+Requires=database@%i.service
+   {% if sonic_asic_platform == 'vs' %}
+Requires=topology.service
+   {% endif %}
+{% else %}
 Requires=database.service updategraph.service
+{% endif %}
 {% if sonic_asic_platform == 'broadcom' %}
 Requires=opennsl-modules.service
 {% elif sonic_asic_platform == 'nephos' %}
 Requires=nps-modules-4.9.0-11-2-amd64.service
 {% endif %}
+{% if multi_instance == 'true' %}
+After=database@%i.service
+After=swss@%i.service
+   {% if sonic_asic_platform == 'vs' %}
+After=topology.service
+   {% endif %}
```

Comment:
The same.

Reply:
Fixed dependency to make sure that single instance and multiple instance dependencies are similar. Topology service is only required in multi-asic VS platform. It is not required in single asic VS platform.